tx.go

```diff
@@ -311,6 +311,15 @@ func (tx Transaction) String() string {
 	return fmt.Sprintf("Transaction{ID: %x}", tx.ID)
 }
 
+func (tx Transaction) Fee() uint64 {
+	fee := uint64(sys.TransactionFeeMultiplier * float64(len(tx.Payload)))
```

Comment:
Rather than looking at the size of the payload, we should look at the entire size of the transaction. This way, we would know by default what the transaction fee is.